cpp/bench/ann/src/raft/raft_benchmark.cu

```diff
@@ -82,6 +79,17 @@ void parse_build_param(const nlohmann::json& conf,
   if (conf.contains("ratio")) { param.kmeans_trainset_fraction = 1.0 / (double)conf.at("ratio"); }
   if (conf.contains("pq_bits")) { param.pq_bits = conf.at("pq_bits"); }
   if (conf.contains("pq_dim")) { param.pq_dim = conf.at("pq_dim"); }
+  if (conf.contains("codebook_kind")) {
+    std::string kind = conf.at("codebook_kind");
```

Comment:
Need to also document all changes to build and search params in the docs/source/ann_benchmarks_param_tuning.md. Otherwise nobody except us know they exist or how to set them.

Reply:
Thanks for reminder, updated! Ideally though, I think, the parameters should match 1-1 the C++ or python API, so that we could refer the user to the corresponding docs.

Reply:
I think it's a great idea to keep the parameter names in line w/ the actual arguments in the APIs but the goal for that parameter tuning guide is to make it painfully easy to use the benchmarks specifically, across all libraries and not just RAFT, so that we're not having the users jump all over the place to run their own benchmarks.